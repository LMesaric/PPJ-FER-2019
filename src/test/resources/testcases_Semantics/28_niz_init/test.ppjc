int main(void) {
    char a[5] = "123";
    const char b[5] = { 'a', 'b', 'c' };
    char c[1025];
    return 0;
}
