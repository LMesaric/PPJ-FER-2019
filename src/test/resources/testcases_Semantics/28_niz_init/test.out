<izravni_deklarator> ::= IDN(4,c) L_UGL_ZAGRADA(4,[) BROJ(4,1025) D_UGL_ZAGRADA(4,])
